documentation/src/main/asciidoc/elasticsearch-integration.asciidoc

```diff
@@ -83,17 +109,137 @@ Other options for the `hibernate.search.elasticsearch.index_management_strategy`
 |CREATE_DELETE|Similarly to 'CREATE' but will also delete the index at shutdown. Commonly used for tests.
 |===============
 
-==== Updating the indexes
+===== Elasticsearch configuration
+
+There is no specific configuration required on the Elasticsearch side.
+
+However there are a few features that would benefit from a few changes:
+
+* analyzers should be defined in the Elasticsearch configuration, see <<elasticsearch-mapping-analyzer>>
+* if you want to retrieve the distance in a geolocation query, install and enable the `lang-groovy` plugin,
+  see <<elasticsearch-query-spatial>>
+* if you want to be able to delete by query, install the `delete-by-query` plugin
```

Comment:
Note that they are deprecating their groovy usage to their own language that should be good enough for what we need. Something to do with security manager constraints.